README.Rmd

```diff
@@ -0,0 +1,102 @@
+---
+output: github_document
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  fig.path = "man/figures/README-",
+  out.width = "100%"
+)
+```
+
+# Calculate the final size of an epidemic <img src="man/figures/logo.png" align="right" width="120"/>
+
+<!-- badges: start -->
+[![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](https://opensource.org/licenses/MIT)
+[![R-CMD-check](https://github.com/epiverse-trace/finalsize/actions/workflows/R-CMD-check.yaml/badge.svg)](https://github.com/epiverse-trace/finalsize/actions/workflows/R-CMD-check.yaml)
+[![Codecov test coverage](https://codecov.io/gh/epiverse-trace/finalsize/branch/main/graph/badge.svg)](https://app.codecov.io/gh/epiverse-trace/finalsize?branch=main)
+<!-- badges: end -->
+
+_finalsize_ provides quick back-of-the-envelope calculations for the final size of an epidemic outbreak in a population with demographic variation in contact patterns, and variation within and between age groups in their susceptibility to disease.
+
+_finalsize_ can help provide rough estimates of the effectiveness of pharmaceutical interventions in the form of immunisation programmes, or the effect of naturally acquired immunity through previous infection (see the vignette).
```

Comment:
Future consideration: I know the vignettes are on a separate branch at the moment, but once they are available you could link to the vignettes here.

Reply:
Definitely - might be a while though :)